### ISSUE TITLE
Login fails with error when debugging on Android Phone

### ISSUE THREAD

**Issue by Steven-FlexTechs** [oktadeveloper/okta-appauth-xamarin-example#2]
Immediately after clicking "Login" the error message appears and nothing else happens. It allows me to keep clicking Logic and keep seeing this error.

![screenshot_20180604-124412](https://user-images.githubusercontent.com/29076250/40935431-5fc006f6-67f5-11e8-8bba-c773fe973deb.png)

I followed the readme instructions. I have no idea how to debug or fix this issue.

Any help would be greatly appreciated.

**Comment by aamirrideo** [comment]
Did you guys implement the Logout feature in IOS, i am struggling to clear the cookies.
Even if i delete the cookies ,browser somehow persists the data and the user is logged in again.

Any inputs would really be helpful.

**Comment by Steven-FlexTechs** [comment]
@aamirrideo I did not, sorry. I stopped at failing to login on Android.

**Comment by nbarbettini** [comment]
@Steven-FlexTechs The error occurs before the browser pops up, right?

IIRC, I had some trouble with this when I tested in Xamarin Live Player, but the Android emulator worked fine.

**Comment by Steven-FlexTechs** [comment]
@nbarbettini Yes, as soon as I click the Login button on that page. Nothing else opens.

I tried the same thing on another computer and it worked that time. This problem seems to be related to Xamarin's environment on my computer.
